src/components/Footer.jsx

```diff
@@ -62,10 +61,7 @@ const Footer = () => {
               <LinkedInIcon className='icon' fontSize='large' />
             </IconButton>
             <IconButton>
-              <InstagramIcon className='icon' fontSize='large' />
-            </IconButton>
-            <IconButton>
-              <TwitterIcon className='icon' fontSize='large' />
+              <FacebookIcon className='icon' fontSize='large' />
```

Comment:
can't you give these temporary links that redirects to respective pages like facebook.com, tweeter.com ?

Reply:
Addressed

Reply:
much obliged